### DISCUSSION TITLE
Lets have a website where beginners can have their first pull request

### DISCUSSION THREAD

**Discussion by JoelAngels** [EddieHubCommunity/support#119] [Ideas]
A simple website made by beginners to get to feel how the pull request works and a lot about github

**Comment by eddiejaoude** [comment]
Great idea 👍 . I can create a community repo we can all contribute to, but would be good to have a clear idea of what it will contain. 

What would it be:
- steps to follow to
   - find a project
   - how to raise an issue
   - make a change
   - raise a PR 
   - ...
- list of possible issues (this could potentially get out of date quickly)
- something else?
